dbt_subprojects/tokens/models/prices/prices_native_tokens.sql

```diff
@@ -41,7 +41,7 @@ FROM
     ('kava-kava', null, 'KAVA', null, null),
     ('kuji-kujira', null, 'KUJI', null, null),
     ('ltc-litecoin', null, 'LTC', null, null),
-    ('matic-polygon', null, 'MATIC', null, null),
```

Comment:
considering prices and the big impact on user queries, think we should add POL here and keep MATIC around for now as well